packages/web/src/components/nav/desktop/LeftNavLink.tsx

```diff
@@ -1,129 +1,53 @@
-import { ComponentProps, useMemo } from 'react'
-
-import { Text, TextProps, useTheme } from '@audius/harmony'
-import { CSSInterpolation } from '@emotion/css'
-import { Interpolation, Theme } from '@emotion/react'
-import { Slot } from '@radix-ui/react-slot'
-import { NavLink, NavLinkProps } from 'react-router-dom'
+import { NavItem, NavItemProps } from '@audius/harmony'
+import { useHistory, useLocation } from 'react-router-dom'
 
 import {
   RestrictionType,
   useRequiresAccountOnClick
 } from 'hooks/useRequiresAccount'
 
-export type LeftNavLinkProps = { disabled?: boolean; asChild?: boolean } & (
-  | Omit<NavLinkProps, 'onDrop'>
-  | Omit<ComponentProps<'div'>, 'onDrop'>
-) & {
-    restriction?: RestrictionType
-  }
+export type LeftNavLinkProps = Omit<NavItemProps, 'isSelected'> & {
+  to?: string
+  disabled?: boolean
+  restriction?: RestrictionType
+}
 
 export const LeftNavLink = (props: LeftNavLinkProps) => {
-  const { asChild, disabled, children, onClick, restriction, ...other } = props
-
-  const theme = useTheme()
-
-  const handleClick = useRequiresAccountOnClick(
-    (e) => onClick?.(e as any),
-    [onClick],
+  const { to, disabled, children, onClick, restriction, ...other } = props
+  const history = useHistory()
+  const location = useLocation()
+
+  const requiresAccountOnClick = useRequiresAccountOnClick(
+    (e) => {
+      onClick?.(e)
+      if (to) {
```

Comment:
let's try to use actual links for this stuff, lmk if you need help getting that set up